hedera-node/hedera-app/src/main/java/com/hedera/node/app/hints/HintsService.java

```diff
@@ -101,11 +116,9 @@ void reconcile(
             @NonNull TssConfig tssConfig);
 
     /**
-     * Returns the verification key for the active hinTS construction, or throws if it is incomplete.
-     * @throws IllegalStateException if the active hinTS construction is incomplete
+     * Stops the hinTS service, causing it to abandon any in-progress work.
      */
-    @NonNull
-    Bytes activeVerificationKeyOrThrow();
+    void stop();
```

Comment:
When will this be called?

Reply:
Whenever platform status changes from `ACTIVE`, we likely want to cancel any scheduled work that would try to gossip `NODE` transactions (and then generate possibly confusing warn/error logs).